kalite/distributed/hbtemplates/video/video-player.handlebars

```diff
@@ -1,6 +1,6 @@
 <div id="video-player"{{#unless video_urls.on_disk }} class="client-online-only"{{/unless}}>
     <video class="video-js vjs-default-skin" id="{{ random_id }}" controls preload="auto" width="100%" height="600" {{#if video_urls.thumbnail }}poster="{{ video_urls.thumbnail }}" {{/if}}data-setup="{}">
-        <source src="{{ video_urls.stream }}" type="{{ video_urls.stream_type }}" />
+        <source src="/content/{{ id }}.mp4" type="video/mp4" />
```

Comment:
the video_urls.stream and video_urls.stream_type  data is empty.
That's why hardcoded to look for that particular url.